server/api/dev/submission-messages.get.ts

```
@@ -0,0 +1,63 @@
+import { cloudflareEnv, jsonResponse } from '~/server/utils/api-response'
+import { queryAll } from '~/server/db'
+
+const enc = new TextEncoder()
+
+function timingSafeEqualText(a: string, b: string): boolean {
+  const left = enc.encode(a)
+  const right = enc.encode(b)
+  if (left.length !== right.length) {
+    let noop = 0
+    for (let i = 0; i < left.length; i += 1) noop |= left[i]!
```

ESLint

[error] 'noop' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)